app/controllers/lifestages_controller.rb

```diff
@@ -1,6 +1,28 @@
 class LifestagesController < FincapTemplatesController
+  attr_accessor :lifestage
```

Comment:
Is the lifestage accessor, accessible outside of the controller? As you have on both resource and show action, can we not use as protected or a private method?